boards/st/stm32mp135f_dk/stm32mp135f_dk_defconfig

```diff
@@ -14,3 +14,6 @@ CONFIG_CONSOLE=y
 
 # UART console (overrides remote proc console)
 CONFIG_UART_CONSOLE=y
+
+# Zephyr Kernel Configuration
+CONFIG_KERNEL_DIRECT_MAP=y
```

Comment:
Should this really be part of the board `_defconfig`?
Doesn't it apply to the series in general ?

Reply:
Right, as we  use ` MMU_REGION_FLAT_ENTRY` in  https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/st/stm32/stm32mp13x/soc.c we need is at Soc level